_posts/2017-03-03-ome-users-meeting-2017.md

```diff
@@ -0,0 +1,23 @@
+layout: post
+title: Speakers announced for OME Users Meeting 2017
+intro-blurb: The OME Consortium are pleased to announce the draft programme for the OME Users Meeting 2017, to be held in Dundee on 31st May to June 2nd.
+---
+Dear All,
+
+The draft programme for the OME Users Meeting 2017 is now available at 
+[www.openmicroscopy.org/site/community/minutes/meetings/12th-annual-users-meeting-2017](https://www.openmicroscopy.org/site/community/minutes/meetings/12th-annual-users-meeting-2017). 
```

Comment:
There's an internal link for this page
 ({{ site.baseurl }}/community/events/12th-annual-users-meeting-2017.html) so you can just link in the text